orcoursetrion/actions/github.py

```diff
@@ -93,6 +94,35 @@ def rerun_studio(course, term, new_term, description=None):
     return repo
 
 
+def release_studio(course, term):
+    """Moves a studio course to be ready for production.
+
+    Currently this will just add a hook to the production server, but
+    it will eventually take care of everything else needed for a
+    transfer as well.
```

Comment:
http://orcoursetrion.readthedocs.org/en/latest/design/#moving-a-course-from-staging-to-production has the details, it is a fairly small divergence for releasing, but still seemed worth splitting in order to keep the conceptual command->action.  This may be DRY overkill here though for just creating a new branch on the xml side, and running the 'Export to Git' API call that doesn't exist on the studio side.